src/options/modules/ManualAdjustments.js

```diff
@@ -0,0 +1,18 @@
+import { getBrowserValue } from "/common/modules/BrowserCompat.js";
+
+/**
+ * Initializes module.
+ *
+ * Applies the adjustments.
+ *
+ * @returns {Promise}
+ */
+export function init() {
+    getBrowserValue({
+        firefox: "https://addons.mozilla.org/firefox/addon/unicodify-text-transformer/?utm_source=unicodify-addon&utm_medium=addon&utm_content=unicodify-addon-settings-inline&utm_campaign=unicodify-addon-settings-inline",
+        thunderbird: "https://addons.thunderbird.net/thunderbird/addon/unicodify-text-transformer/?utm_source=unicodify-addon&utm_medium=addon&utm_content=unicodify-addon-settings-inline&utm_campaign=unicodify-addon-settings-inline",
+        chrome: "https://chrome.google.com/webstore/detail/unicodify-text-transformer/",
```

Comment:
```suggestion
        firefox: "https://addons.mozilla.org/firefox/addon/unicodify-text-transformer/?utm_source=awesomeEmojiPicker-addon&utm_medium=addon&utm_content=awesomeEmojiPicker-addon-settings-inline&utm_campaign=awesomeEmojiPicker-addon-settings-inline",
        thunderbird: "https://addons.thunderbird.net/thunderbird/addon/unicodify-text-transformer/?utm_source=awesomeEmojiPicker-addon&utm_medium=addon&utm_content=awesomeEmojiPicker-addon-settings-inline&utm_campaign=awesomeEmojiPicker-addon-settings-inline",
        chrome: "https://chrome.google.com/webstore/detail/unicodify-text-transformer/?utm_source=awesomeEmojiPicker-addon&utm_medium=addon&utm_content=awesomeEmojiPicker-addon-settings-inline&utm_campaign=awesomeEmojiPicker-addon-settings-inline",
```

Note: The utm _source_ etc. should be the referrer/(well) source where we get from, not the target we get to. As this is for the statistics on the AMO page.